### DISCUSSION TITLE
Looking for Podman Static Binaries

### DISCUSSION THREAD

**Discussion by SatyaVaraprasad1978** [containers/podman#10353] [Q&A]
Hi ,

Please may I know where I can find the latest version Podman Static Binaries with steps to setup for Redhat/CentOs Platform. 

Regards,
Satya

**Comment by rhatdan** [comment]
I think the static binaries are available in the NIX builds.  But it is not that easy.  Podman requires particular versions of other packages like fuse-overlayfs, slirp4netns, container-selinux, crun, runc, conmon ...

So just getting a statically linked version of Podman may not be enough to get it running on centos 7 and rhel7 if that is your goal.  If you want the packaged versions for RHEL8 and CEntos8, they should be getting updated to 3.0 within the next couple of weeks.
